scripts/check-i18n.js

```
@@ -0,0 +1,561 @@
+    /^[a-z]+$/i, // Single format token like "short", "long" (for Intl.DateTimeFormat)
+  ];
+  return dateFormatPatterns.some((pattern) => pattern.test(trimmed));
+};
+
+// Check if text looks like a regular expression pattern
+const looksLikeRegexPattern = (text) => {
+  const trimmed = text.trim();
+  // Common regex patterns (character classes, quantifiers, anchors, etc.)
+  if (/^[.*+?^${}()|[\]\\\/\-]+$/.test(trimmed)) return true;
+  // Patterns with common regex syntax
+  if (/^[^]*[\^$.*+?{}()[\]|\\][^]*$/.test(trimmed)) return true;
```

Biome (2.1.2)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)